src/validation/error.messages.ts

```diff
@@ -35,6 +35,8 @@ export enum ErrorMessages {
   LEGAL_ENTITY_BO_NAME = "Enter its name",
   ENTITY_CORRESPONDENCE_ADDRESS = "Enter their correspondence address",
   START_DATE_MUST_BE_AFTER_DOB = "Start date must be on or after date of birth",
+  UNABLE_TO_RETRIEVE_ENTITY_NUMBER = "Unable to retrieve entity number",
```

Comment:
Might be difficult to test/replicate but these new user error messages should get a mention on the JIRA ticket.